openlibrary/templates/account/create.html

```diff
@@ -72,8 +72,8 @@ <h1>$_("Sign Up")</h1>
         $if form.has_recaptcha:
             <div class="formElement">
                 <div class="label">$_("Please type in the text or number(s) below.")<br/><span class="smaller lighter">$_("If you have security settings or privacy blockers installed, please disable them to see the reCAPTCHA.")</span></div>
+                <div class="g-recaptcha" data-sitekey="{{$form['recaptcha'].public_key}}"></div>
```

Comment:
We don't use jinja2, i.e. `{{}}` syntax. The `$` variable is telling python to render everything after it (i.e. `form['recaptcha'].public_key`). The `{{` and `}}` need to be removed.

You can verify this by inspecting the `data-sitekey` param within the registration page source.